assets/js/react-shim.test.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * React Shim tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import * as react from 'react';
+
+/**
+ * Internal dependencies
+ */
+import * as reactShim from './react-shim';
+
+describe( 'react shim', () => {
+	it( 'mirrors all exports from the react package', () => {
+		const realExports = Object.keys( react ).sort();
+		const shimExports = Object.keys( reactShim ).sort();
+
+		expect( shimExports ).toEqual( realExports );
+	} );
```

Comment:
Unlike the test for `element-shim` this should also test that the default exports match as well.

Reply:
Oh, true, I'll add that 😓

Reply:
Added!